.circleci/config.yml

```diff
@@ -52,7 +52,7 @@ jobs:
           name: armada bundle
```

Comment:
changes in this file are because CRA built to `build`, while Vite builds to `dist`. we could make Vite build to `build` if these changes are too much of a hassle

Reply:
np